src/libpriv/rpmostree-origin.c

```diff
@@ -407,6 +435,24 @@ rpmostree_origin_set_jigdo_version (RpmOstreeOrigin *origin,
   origin->cached_jigdo_version = g_strdup (version);
 }
 
+/* The jigdoRPM is highly special; it doesn't live in the rpmdb for example, as
+ * that would be fully circular. Yet, it's of critical importance to the whole
```

Comment:
That's a good point I hadn't really thought through. It does feel odd that it's *not* in the rpmdb, but that's clearly by design.